modules/exploits/windows/local/tokenmagic.rb

```diff
@@ -0,0 +1,228 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core/post/windows/powershell'
```

Comment:
This is certainly required?

Reply:
Tested without, module runs fine, removing.